Dockerfile

```diff
@@ -1,12 +1,8 @@
 FROM jujusolutions/jujubox:latest
+MAINTAINER Charles Butler <charles.butler@canonical.com>
 
-VOLUME "/home/ubuntu/.juju"
-VOLUME "/home/ubuntu/trusty"
-VOLUME "/home/ubuntu/builds"
-VOLUME "/home/ubuntu/layers"
-VOLUME "/home/ubuntu/interfaces"
-ADD install-review-tools.sh /install-review-tools.sh
-RUN /install-review-tools.sh
-# Override jujubox run.sh
-ADD run.sh /run.sh
-CMD /run.sh
+ENV LAYER_PATH=/home/ubuntu/charms/layers
+ENV INTERFACE_PATH=/home/ubuntu/charms/interfaces
+
```

Comment:
Builds is a sub-directory of $JUJU_REPOSITORY which you're already exposing as a volume and an ENV in jujubox.  I don't think we need an explicit directive here unless I'm mistaken